ci/Jenkinsfile

```diff
@@ -5,8 +5,8 @@ def HOMEgfs = 'none'
 def CI_CASES = ''
 def GH = 'none'
 // Location of the custom workspaces for each machine in the CI system.  They are persistent for each iteration of the PR.
-def NodeName = [hera: 'Hera-EMC', orion: 'Orion-EMC', hercules: 'Hercules-EMC', gaea: 'Gaea', gaeac6: 'Gaeac6-EMC']
-def custom_workspace = [hera: '/scratch1/NCEPDEV/global/CI', orion: '/work2/noaa/stmp/CI/ORION', hercules: '/work2/noaa/global/CI/HERCULES', gaea: '/gpfs/f5/epic/proj-shared/global/CI', gaeac6: '/gpfs/f6/drsa-precip3/proj-shared/global/CI']
+def NodeName = [hera: 'Hera-EMC', orion: 'Orion-EMC', hercules: 'Hercules-EMC', gaeac5: 'gaeac5', gaeac6: 'Gaeac6-EMC']
```

Comment:
The map from MACHINE_ID to Node Name is acctualy gaeac5 to GaeaC5.  I have an update for this file that also address the "-" in the Node Names.  I will make it PR to your branch here.